src/mjolnir/admin.cc

```diff
@@ -10,6 +10,8 @@
 namespace valhalla {
 namespace mjolnir {
 
+using polygon_type = boost::geometry::model::polygon<point_type>;
```

Comment:
this looks unused

Reply:
Thanks. I added that for an early change where I tested if the tile's bounding box was totally inside the 1 state query resulting multi-polygon. If so I was going to just return that one admin and then set tile_withing_one_admin within graphbuilder.cc. This would have removed the need to do point in poly for all other nodes within the tile. Unfortunately this led to a mismatch in tile sizes - so some admin information must have changed. I will leave this for a later (possible) update.